### PR TITLE
spanish translation added and some minor changes in the README file

### PR DESCRIPTION
**Pull request by EuriNaiz**
In the original README I just deleted Inc after Free Software Foundation, fixed <br> tags, and removed a feature mentioned twice.
The README.ES.MD is just the translation.

### CODE REVIEW

**Comment by Sangwan5688** [comment]
Can you just add a language switch option too. Everything else looks fine

File: README.ES.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+flutter pub get
+flutter run
+```
+
+4. (Opcional) Puede configurar su propia base de datos para la aplicación o utilizar la predeterminada. Para configurar su propia base de datos, siga [éstas instrucciones](https://github.com/Sangwan5688/BlackHole/wiki/Set-up-your-own-Firebase-Database).
+
+## Contribuciones
+
+Las contribuciones son bienveenidas, Por favor lea los [lineamientos de contribución](https://github.com/Sangwan5688/BlackHole/blob/main/CONTRIBUTING.md) antes de contribuir.
+
```
Comment (Sangwan5688) [inline README.ES.md:74]: ## Facing any Issue?

Have a look at some [common Issues](https://github.com/Sangwan5688/BlackHole/wiki/Common-Issues) that you might face. If still face any issue feel free to open an Issue :)

File: README.md [hunk 2]
```diff
@@ -40,36 +39,36 @@ Awesome Music Player made with Flutter!
...
 flutter pub get
 flutter run
 ```
+
 4. (Optional) You can setup your own database for the app or use the default. To setup your own follow [these instructions](https://github.com/Sangwan5688/BlackHole/wiki/Set-up-your-own-Firebase-Database).
 
-## Contribute 
+## Contribute
 
 Contributions are welcomed. Please read our [contributing guidelines](https://github.com/Sangwan5688/BlackHole/blob/main/CONTRIBUTING.md) before contributing.
```
Comment (Sangwan5688) [inline README.md:74]: ## Facing any Issue?

Have a look at some [common Issues](https://github.com/Sangwan5688/BlackHole/wiki/Common-Issues) that you might face. If still face any issue feel free to open an Issue :)

**Review by EuriNaiz — COMMENTED** [review]
Added the option to switch languages.

**Comment by Sangwan5688** [comment]
I commented in a section yesterday. There a new section in the README, "Facing any Issue?". Just add it in both the files. Everything else looks fine.

**Review by Sangwan5688 — APPROVED** [review]

**Pull request merged**